### PR TITLE
Updated greek translation

### PR DESCRIPTION
**Pull request by manussos**
Following the instructions for translators, as given in the `readme.md` file, a current Greek translation is submited.

### CODE REVIEW

File: data/com.github.jeromerobert.pdfarranger.metainfo.xml [hunk 1]
```diff
@@ -37,6 +38,15 @@ See file COPYING or go to <http://www.gnu.org/licenses/> for full license detail
       It is a frontend for pikepdf.
     </p>
 
+    <p xml:lang="es">
```
Comment (angsch) [inline data/com.github.jeromerobert.pdfarranger.metainfo.xml:41]: "es" -> "el" (same for line 46)

**Comment by angsch** [comment]
Thank you very much for your PR. Could you change the language code for metainfo.xml?

**Comment by manussos** [comment]
Whoops! I've just changed it.
Thanks for the catch!

File: po/el.po [hunk 2]
```diff
@@ -1,506 +1,1370 @@
...
-msgstr ""
+msgstr "Απαιτείται κωδικός"
 
-#: pdfarranger/core.py:186
+#: pdfarranger/core.py:202
 msgid "The password will be remembered until you close PDF Arranger."
-msgstr ""
-
-#: pdfarranger/core.py:187
+<<<<<<< Updated upstream
```
Comment (dreua) [inline po/el.po:62]: These are leftovers from a git merge/rebase conflict. Those need to be cleaned up before we can move on with this.

**Review by dreua — CHANGES_REQUESTED** [review]

File: po/el.po (hunk 2 above)
Comment (dreua) [reply po/el.po:62]: This should also fix CI. I'm not sure how you ended up with this situation, tbh, but I know Git can be confusing at the beginning. Let me know if you need any assistance :wink:

Comment (manussos) [reply po/el.po:62]: Yeah, working with git is a joyful experience 😵‍💫
No idea how I got it messed up like that!

Comment (manussos) [reply po/el.po]: Should be okay now, hopefully 🤞...

Comment (dreua) [reply po/el.po]: It becomes joyful (or at least useful) when you get used to Git :wink:

**Review by dreua — APPROVED** [review]
Looks good to me now, thank you for your contribution!

**Comment by manussos** [comment]
You are welcome @dreua. Thank you for the merge and for taking the time to review!

**Pull request merged**